lit_parrot/model.py

```diff
@@ -49,6 +49,11 @@ def _init_weights(self, module: nn.Module) -> None:
             # https://huggingface.co/stabilityai/stablelm-base-alpha-3b/blob/main/config.json#L12
             module.eps = 1e-5
 
+    def reset_cache(self) -> None:
+        self.rope_cache = None
+        self.mask_cache = None
```

Comment:
Why reset these 2? They don't depend on the batch size or input sequence length

Reply:
Right, and block size doesn't change across generate, I removed them.